08-rails/common-view-helpers.md

```diff
@@ -45,6 +45,62 @@ The above `image_tag` will give you this HTML:
 ```html
 <img src="/assets/cat.jpg" alt="The cutest cat in the whole world.">
 ```
+
+## Using Named Routes
+
+Above we used hard-coded paths like "/profile" in our helpers.  We can also use methods like `link_to` with named routes from our `config/routes.rb` file.  
```

Comment:
Would it be worth reiterating here why we would want to use the named route helpers in our views?